src/ui/ui_settings/ui_settings_service.js

```diff
@@ -190,6 +189,19 @@ export class UiSettingsService {
       const resp = await this._savedObjectsClient.get(this._type, this._id);
       return resp.attributes;
     } catch (error) {
+      if (isNotFoundError(error)) {
+        try {
+          await createOrUpgradeSavedConfig({
+            savedObjectsClient: this._savedObjectsClient,
+            version: this._id,
+            buildNum: this._buildNum,
+            log: this._log,
+          });
+        } catch (e) {
```

Comment:
We should really be checking this error to make sure it's what we expect. Otherwise we could be swallowing errors that indicate unexpected issues down the road.